lib/ApiResource.php

```diff
@@ -151,6 +158,18 @@ protected static function _create($params = null, $options = null)
         return $obj;
     }
 
+    protected static function _update($id, $params = null, $options = null)
```

Comment:
Just in the interest of leaving things better than we found them, mind putting some documentation on this function?